include/revng/Yield/Calls/Slices.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Yield/Graph.h"
+
+class MetaAddress;
+
+namespace yield::calls {
+
+Graph vettedSlice(const Graph &Input);
```

Comment:
* *vetted*?
* What's the slicing for? Can you add some docs? Sounds interesting.

Reply:
Sorry, I changed the name in later commit 😅 

I guess this git tree need a bit more love than I anticipated.

1. Kind of like "filtered" but more in a sense that the edges that remain are deliberately selected.
2. Sure, but only for the latest version :wink: